internal/message_tests_protolegacy_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020-2024 Buf Technologies, Inc.
```

Comment:
Did you mean for this file to be named `message_sets_protolegacy_test.go` to match the naming of the other test?

Reply:
done